### DISCUSSION TITLE
Alpine Nextcloud Hub - Login

### DISCUSSION THREAD

**Discussion by ucramos** [tteck/Proxmox#1906] [Help Needed] [answered]
Hello,

First of all i just want to say that these scripts are amazing. A total linux noob was already able to get 4 VM/LXC up and running!

I´ve installed Alpine Nextcloud Hub. The script runs great.
After conclusion, when accessing the website, i get a login window.
I was not asked to register a user or something like that.
Was there an error on the installation?
Or i haven´t seen the default user/pass?

I´ve run the script 3 times, default and personalize and the behavior is the same.

**Comment by tteck** [comment, marked as answer]
![Screenshot 2023-10-10 7 32 59 PM](https://github.com/tteck/Proxmox/assets/21342670/1402d328-31e2-4b83-ae4c-56eaabc3b54f)

**Comment by ucramos** [comment]
Thank you for your quick reply.

I don´t know why the scripts run inside the LXC was not working. Only after i rebooted PROXMOX it ran...
Thank you again.
